monkey/monkey_island/cc/services/attack/technique_reports/T1075.py

```diff
@@ -35,9 +35,9 @@ def get_report_data():
         successful_logins = list(mongo.db.telemetry.aggregate(T1075.query))
         data.update({'successful_logins': successful_logins})
         if successful_logins:
```

Comment:
set status according to successful_logins and count_documents(query),
afterwards use only 1 call to get_message_and_status